tests/src/test/java/org/eclipse/hono/tests/registry/DeviceRegistrationMandatoryAmqpIT.java

```diff
@@ -0,0 +1,144 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.tests.registry;
+
+import io.vertx.core.Vertx;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.proton.ProtonClientOptions;
+import org.eclipse.hono.client.HonoClient;
+import org.eclipse.hono.client.ServiceInvocationException;
+import org.eclipse.hono.tests.IntegrationTestSupport;
+import org.eclipse.hono.util.Constants;
+import org.eclipse.hono.util.RegistrationConstants;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+
+import java.net.HttpURLConnection;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Tests verifying the behavior of the Device Registry component's Device Registration AMQP endpoint.
+ */
+@RunWith(VertxUnitRunner.class)
+public class DeviceRegistrationMandatoryAmqpIT {
```

Comment:
I would rather have the classes named the other way around, i.e. have the mandatory tests in `DeviceRegistrationAmqpIT` and the optional ones in `DeviceRegistrationOptionalAmqpIT`. This way, the optional tests become the *special* case which is more appropriate FMPOV.

Reply:
make sense to me !